app/com/linkedin/drelephant/spark/SparkMetricsAggregator.scala

```diff
@@ -78,27 +84,32 @@ class SparkMetricsAggregator(private val aggregatorConfigurationData: Aggregator
   }
 
   //calculates the resource usage by summing up the resources used per executor
-  private def calculateResourceUsage(executorSummaries: Seq[ExecutorSummary], executorMemoryBytes: Long): (BigInt, BigInt) = {
+  private def calculateResourceUsage(data: SparkApplicationData): (BigInt, BigInt) = {
```

Comment:
Since this method is private how are u testing in unit test . Since this is important method , can we have make it visible for testing (if not already covered in unit test case)

Reply:
getResult method is public, which gives all the value computed by calculateResourceUsage method. Which is covered in test SparkMetricsAggregatorTest.